cirq/devices/noise_model.py

```diff
@@ -151,6 +153,14 @@ def noisy_operation(self, operation: 'cirq.Operation') -> 'cirq.OP_TREE':
             noisy version of the given operation.
         """
 
+    @abc.abstractmethod
+    def is_mixture_of_unitaries(self) -> bool:
```

Comment:
We can't introduce new abstract methods like this. It will break existing implementations.

We could refactor into

`definitely_only_adds_mixture_operations`

and

`definitely_only_adds_unitary_operations`

with default implementations returning `False`.